content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+
+## Act on registrations and assignments in CI/CD
+
+GTO [uses Git tags] to register artifact versions and assign them to stages.
+This means we can act upon these operations in any Git-based system such as many
+CI/CD platforms.
+
+[uses git tags]: /doc/gto/user-guide#git-tags-format
+
+<details>
+
+### Click to set up a working repository to try this.
+
+Let's fork the [example-gto repo](https://github.com/iterative/example-gto) repo
+(you'll need a [GitHub](https://github.com/signup) account first). For CI/CD to
+start, you'll need to enable it on the "Actions" page of your fork.
+
+1. [Fork the repo](https://github.com/iterative/example-gto/fork). Make sure you
+   uncheck "Copy the `main` branch only" to copy Git tags as well:
+   <img width="877" alt="image" src="https://user-images.githubusercontent.com/6797716/199275275-439335f4-6f54-4cd7-910d-fc29ad3c095c.png">
+
+2. Enable workflows in your repo for a new Git tag to trigger CI:
+   <img width="869" alt="image" src="https://user-images.githubusercontent.com/6797716/199272682-dfd628bf-9599-4e85-a623-bf4a10c3d7e1.png">
+
+</details>
+
+Now let's push the Git tags we created locally to your repository (don't forget
+to replace `aguschin` with your username):
+
+```cli
+$ git remote update origin https://github.com/aguschin/example-gto
+$ git push origin cv-class@v0.1.14 cv-class#dev#1
+ * [new tag]         cv-class@v0.1.14 -> cv-class@v0.1.14
+ * [new tag]         cv-class#dev#1 -> cv-class#dev#1
+```
+
+<details>
+
+### Or just repeat the assignment for your new repo
+
+We can do the same thing we did locally, but for your remote repo (don't forget
+to replace `aguschin` with your username):
+
+```cli
+# since we didn't register a version on remote, GTO will do that for us:
+$ gto assign cv-class --stage dev \
+    --repo https://github.com/aguschin/example-gto
+Created git tag 'cv-class@v0.1.14' that registers a version
+Running `git push origin cv-class@v0.1.14`
+Successfully pushed git tag cv-class@v0.1.14 on remote.
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.14'
+Running `git push origin cv-class#dev#1`
+Successfully pushed git tag cv-class#dev#1 on remote.
+```
+
+</details>
+
+Git tags pushed trigger the
+[CI workflow](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml):
+
+<img width="875" alt="image" src="https://user-images.githubusercontent.com/6797716/199276636-bf996ad3-7d9c-4100-9f3c-6444730e4d19.png">
+
+"GTO: figure out what was registered/promoted" step interprets the Git tag that
+triggered the workflow and passes the information to the next steps. The
+information is used later to trigger "Publish" or "Deploy" steps (since the CI
+on the screenshot was triggered by `cv-class#dev#1` Git tag, the "Publish" step
+was skipped).
+
+If you want to see CI examples without reproducing this Get Started, you can
+check out them in
+[the example-repo](https://github.com/iterative/example-gto/actions).
```

Comment:
We can simplify this quite a bit if the above suggestions are committed. I'd also remove the image, leaving a link to the example repo workflows instead (but up to you on images):

```suggestion
<admon type="info">

To see what the example repo's [CI/CD jobs] look like, see its [GitHub Actions] page.

[ci/cd jobs]:
  https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml
[github actions]: https://github.com/iterative/example-gto/actions

</admon>
```